android/anc/src/main/java/org/smartregister/fhircore/anc/ui/family/register/components/FamilyListItem.kt

```diff
@@ -136,7 +136,7 @@ fun servicesCard(modifier: Modifier, text: String, color: Color) {
 @Composable
 @Preview(showBackground = true)
 fun FamilyRowPreview() {
-  val fmi = FamilyMemberItem("fm1", "21", "F", true)
+  val fmi = FamilyMemberItem("fmname", "fm1", "21", "F", true)
 
   val familyItem =
     FamilyItem("fid", "Name ", "M", "27", "Nairobi", true, listOf(fmi, fmi, fmi), 4, 5)
```

Comment:
how are these used, is there a reason these need to be hard coded?

Reply:
it's just being used for preview in the editor

Reply:
k cool, sorry I'm not that familiar with this, what does that mean?

Reply:
on designing a UI in we need to see how the UI will look on an actual device/emulator so, for that purpose Android Studio (IDE) provides a layout preview window. here is an example https://developer.android.com/jetpack/compose/tooling

Reply:
got ok sounds good, thank you for explaining